shell/app/node_main.cc

```diff
@@ -165,7 +184,15 @@ int NodeMain(int argc, char* argv[]) {
       versions.SetReadOnly(ELECTRON_PROJECT_NAME, ELECTRON_VERSION_STRING);
     }
 
-    node::LoadEnvironment(env);
+    // TODO(codebytere): we should try to handle this upstream.
+    {
+      node::InternalCallbackScope callback_scope(
+          env, v8::Local<v8::Object>(), {1, 0},
+          node::InternalCallbackScope::kAllowEmptyResource |
+              node::InternalCallbackScope::kSkipAsyncHooks);
```

Comment:
Is it recommended to skip async hooks in callback scope ? In that case should we also do that for our node integration https://github.com/electron/electron/blob/master/shell/common/api/electron_bindings.cc#L136 ?

Reply:
/cc @codebytere